mindsdb/integrations/handlers/gcs_handler/requirements.txt

```diff
@@ -0,0 +1,5 @@
+duckdb
```

Comment:
Let's remove this from the list here as it is already part of the main dependencies for MindsDB. It is causing the CI to fail.

Reply:
Done